lib/features.js

```diff
@@ -66,6 +66,9 @@ module.exports = {
     // enable parsing of shorthand object literal properties
     objectLiteralShorthandProperties: false,
 
+    // Allow duplicate object literal properties (except '__proto__')
+    duplicateObjectLiteralProperties: false,
+
```

Comment:
I think it would be better to name `objectLiteralDuplicateProperties` to match the other flags that apply to object literals.